packages/strapi-design-system/src/SimpleMenu/SimpleMenu.js

```diff
@@ -0,0 +1,111 @@
+import React, { useRef, useState, Children, cloneElement } from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import FilterDropdown from '@strapi/icons/FilterDropdown';
+import { TextButton, Text } from '../Text';
+import { Box } from '../Box';
+import { Row } from '../Row';
+import { Popover } from '../Popover';
+import { getOptionStyle } from './utils';
+
+const OptionButton = styled.button`
+  border: none;
+  padding: 0;
+  background: transparent;
+  cursor: pointer;
+  ${getOptionStyle}
+`;
+const OptionLink = styled.a`
+  text-decoration: none;
+  ${getOptionStyle}
+`;
+const MenuButton = styled.button`
+  border: none;
+  background: transparent;
+  display: flex;
+  align-items: center;
+  font-size: ${12 / 16}rem;
+  svg {
+    height: 4px;
+  }
+`;
+
+export const MenuItem = ({ children, onClick, href, ...props }) => {
+  if (href) {
+    return (
+      <Row as="li" justyfiContent="center">
+        <OptionLink href={href} {...props}>
+          <Box padding={2}>
+            <Text>{children}</Text>
+          </Box>
+        </OptionLink>
+      </Row>
+    );
+  }
+
+  return (
+    <Row as="li" justyfiContent="center">
+      <OptionButton onClick={onClick} {...props}>
+        <Box padding={2}>
+          <Text>{children}</Text>
+        </Box>
+      </OptionButton>
+    </Row>
+  );
+};
+MenuItem.defaultProps = {
+  onClick: () => {},
+  href: undefined,
+};
+MenuItem.propTypes = {
+  children: PropTypes.string.isRequired,
+  href: PropTypes.string,
+  onClick: PropTypes.func,
+};
+
+export const SimpleMenu = ({ label, children, ...props }) => {
+  const menuButtonRef = useRef();
+  const [visible, setVisible] = useState(false);
+
+  const handleBlur = (e) => {
+    if (!e.currentTarget.contains(e.relatedTarget)) {
+      setVisible(false);
+    }
+  };
+
+  const childrenClone = Children.toArray(children).map((child) =>
+    cloneElement(child, {
+      onClick: () => {
+        setVisible(false);
+        child.props.onClick();
+      },
+    }),
+  );
+
+  return (
+    <div>
+      <MenuButton {...props} onClick={() => setVisible((s) => !s)} ref={menuButtonRef}>
```

Comment:
You can also add `aria-haspopup={true}` to inform the user that this button aims to open a popup 😊 

Also, you can use `aria-expanded={visible}` so that the user knows in which state is the menu

Reply:
And you can also add `aria-controls={menuId}` so that assistive technologies can create a relation between the element that aims to open the popup and the popup itself